### ISSUE TITLE
To support coverage-report for @playwright/experimental-ct-react

### ISSUE THREAD

**Issue by hsuehic** [bgotink/playwright-coverage#14]
Hi @bgotink.  Do you have a plan to support coverage report for @playwright/experimental-ct-react.

**Comment by bgotink** [comment]
Nope, I'm not planning on extending this package at the moment, I don't use react in any project, and I haven't looked at component testing in playwright yet.

It might not be possible to support this, depending on how the sourcemaps are handled in these mounted components and on the timings of when these sourcemaps become unavailable. We know from #13 that the test server has to be running long enough for the reporter to access the sourcemaps, as we can't currently make playwright wait before shutting off the server.
